packages/sync-actions/src/utils/copy-empty-array-props.js

```diff
@@ -15,13 +15,49 @@ export default function copyEmptyArrayProps(oldObj = {}, newObj = {}) {
           ...newObj,
           ...nextObject,
         }
+
+        if (key === 'custom') {
+          return {
+            ...merged,
+          }
+        }
+
+        if (Array.isArray(value) && newObj[key] && newObj[key].length >= 1) {
```

Comment:
Just wondering, should i add comments, to explain whats going on, or is the code explains itself?!

Reply:
your question already answered the question 😉 
if this introduce an ounce of cognitive overload, then absolutely.